assets/app/scripts/controllers/overview.js

```diff
@@ -12,9 +12,7 @@ angular.module('openshiftConsole')
     $scope.pods = {};
```

Comment:
Wondering if a more consistent naming scheme would help this be more understandable (it's possible the answer is no).

Proposed naming template: <object>[By<FirstKey>][By<SecondKey>]

Things that currently don't follow that pattern:

```
deploymentPods -> podsByDeployment
servicePodsNoDeployment -> podsByServiceByDeployment
serviceDeploymentsByDeploymentConfig -> deploymentsByServiceByDeploymentConfig
serviceDeploymentConfigs -> deploymentConfigsByService
```

Also, could empty keys be used to represent things outside of services/deployments, etc?

Pods in deployments under services: `podsByServiceByDeployment["serviceid"]["deploymentid"]`
Pods not in deployments under services: `podsByServiceByDeployment["serviceid"][""]`
Pods in deployments not under services: `podsByServiceByDeployment[""]["deploymentid"]`
Pods not in deployments not under services: `podsByServiceByDeployment[""][""]`

Not sure if this makes things easier or harder, just an idea